moped-editor/src/components/GridTable/Search.js

```diff
@@ -99,7 +99,8 @@ const Search = ({
     setFilters({});
     setIsOr(false);
     setSearchParams((prevSearchParams) => {
-      prevSearchParams.delete("filters");
+      prevSearchParams.delete("filter");
+      prevSearchParams.delete("isOr");
       return prevSearchParams;
```

Comment:
cool—i see this mirrors the same behavior as when resetting the filters via the `handleClearFilters` function.